android/src/main/java/com/audiowaveform/AudioPlayer.kt

```diff
@@ -151,6 +158,23 @@ class AudioPlayer(
         }
     }
 
+    fun setPlaybackSpeed(speed: Float?, promise: Promise) {
+        try {
+            if (speed != null && speed > 0f) {
+                // Set the playback speed on the player
+                val playbackParameters = player.playbackParameters.withSpeed(speed)
+                player.playbackParameters = playbackParameters
+                promise.resolve(true)
+            } else {
+                // Reject the promise if the speed is invalid
+                promise.reject("Invalid speed", "Playback speed must be a positive value.")
+            }
```

Comment:
we set check if speed is negative or non-positive set to default speed.
```suggestion
           val playbackSpeed = if (speed != null && speed > 0f) speed else 1f
val playbackParameters = player.playbackParameters.withSpeed(playbackSpeed)
player.playbackParameters = playbackParameters
```